public/app/features/alerting/unified/Analytics.ts

```diff
@@ -39,3 +39,16 @@ export function withPerformanceLogging<TFunc extends (...args: any[]) => Promise
     return response;
   };
 }
+
+export const trackAlertRuleCreation = (props: AlertRuleTrackingProps) => {
+  reportInteraction('grafana_alerting_rule_creation', props);
+};
+
+export const trackAlertRuleAborted = (props: AlertRuleTrackingProps) => {
```

Comment:
I wonder if Is there a way to connect create/abort action so in the future we could for example analyse the time between these events

Reply:
Good point. We don't have a rule identifier as it's not created yet but I'm sending the user id. I think we can connect them using that value as we'll be only taking into account the first trackings.